python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -430,10 +433,27 @@ class _ActorPool:
     This class is in charge of tracking the number of in-flight tasks per actor,
     providing the least heavily loaded actor to the operator, and killing idle
     actors when the operator is done submitting work to the pool.
+
+    Args:
+        max_tasks_in_flight: The maximum number of tasks to queue on a single actor at
+            a time. The actual max_tasks_in_flight starts at 1, and then ramps up
+            to this value after `ramp_up_threshold` actors have been created, to allow
+            for even scheduling for datasets with a small number of partitions.
+        max_tasks_rampup_threshold: The number of actors to wait for before ramping up
```

Comment:
This seems exactly the semantics of `AutoscalingConfig.min_workers`? This I think we already honor it by creating min_workers actors: https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/execution/operators/actor_pool_map_operator.py#L87

Reply:
We start the actors, but we don't actually wait for them to go from pending to running.

In this case, there was a 1 ms delta in when all the min workers actually were created. So we immediately fill up the first 4 actors with more tasks than they can handle, while the last 4 are just idle.